BrushedMotorControllerFirmware/CMakeLists.txt

```
@@ -5,7 +5,7 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(MotorController)
 
 target_include_directories(app PRIVATE
-     "src" "include"
+     "src" "include/uart_shell" "include"
```

```suggestion
     "src" "include"
```

and use `#include "uart_shell/shell_utils.h"` etc